platform/engineering/design-docs/2020-04-09-separate-content-build.md

```diff
@@ -232,8 +223,6 @@ To ensure that one build won't override files in the other, we'll have to
   tests can run a browser and access the applications
 - **Important note:** These HTML files will not be served in production; the
   landing pages for the applications come from the content build
-  - To ensure these pages aren't made in production, the deploy script will copy
-    only the files in `/applications/`
```

Comment:
I think this part is still applicable, but less important with the separated buckets. We still don't want to upload the fake landing pages to S3, I think.